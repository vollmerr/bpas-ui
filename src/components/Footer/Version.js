import React from 'react';
import { Link } from 'react-router-dom';
import styled from 'styled-components';
import theme from '../theme';

const Wrapper = styled.div`
  flex: 3;
  text-align: right;
  padding: ${theme.size.xs}px ${theme.size.sm}px;
  white-space: nowrap;
  color: ${theme.color.primaryInverse};
`;

function Version() {
  return (
    <Wrapper>Version {process.env.VERSION}</Wrapper>
  );
}

export default Version;
