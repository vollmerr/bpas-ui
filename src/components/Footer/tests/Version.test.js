import React from 'react';
import { shallow } from 'enzyme';
import Version from '../Version';

process.env.VERSION = 'test 123';

describe('<Version />', () => {
  it('renders correctly', () => {
    const tree = shallow(
      <Version />
    );
    expect(tree).toMatchSnapshot();
  });
});
