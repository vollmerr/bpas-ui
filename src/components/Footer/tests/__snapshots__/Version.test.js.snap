// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Version /> renders correctly 1`] = `
<styled.div>
  Version 
  test 123
</styled.div>
`;
